models/topics.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection.js');
+
+exports.fetchTopics = () => {
+    return db.query(
+        `
+        SELECT * FROM topics;
+        `
+    ).then((result) => {
+        if(result.rows.length === 0){
+            return Promise.reject({
+                status: 404,
+                msg: 'topics not found',
+            });
```

Comment:
This error handling never needs to be here Diana - there will never be a case where this would trigger